src/json-helper.c

```diff
@@ -27,6 +27,51 @@
 #include "json-helper.h"
 #include <stddef.h>
 
+
+/**
+ * @brief Get the first JsonNode element matching path in json_node.
+ *
+ * @param[in]  json_node JsonNode to query
+ * @param[in]  path      Query path
+ * @param[out] error     Error
+ * @return JsonNode*, matching JsonNode element (must be freed), NULL on error
+ */
+static JsonNode* json_get_first_matching_element(JsonNode *json_node,
```

Comment:
Note that introducing a static non-used function will trigger a warning message for this commit. Maybe ok if not testing with -Werror on.

Reply:
True, but that shouldn't be a problem. I'd like to keep the commits separated as they are now.

Reply:
Ok